add-on/panels/js/conversation.jsx

```diff
@@ -150,14 +149,11 @@ loop.conversation = (function(mozL10n) {
         }
       });
 
-      // We want data channels only if the text chat preference is enabled.
-      var useDataChannels = results[++requestIdx];
 
       var dispatcher = new loop.Dispatcher();
       var sdkDriver = new loop.OTSdkDriver({
         constants: constants,
         isDesktop: true,
-        useDataChannels: useDataChannels,
```

Comment:
You need to keep this line but change it to:

    useDataChannels: true

Otherwise the text chat doesn't get enabled.